src/internal/config.ts

```diff
@@ -1,11 +1,28 @@
 import { Subscriber } from './Subscriber';
 import { ObservableNotification } from './types';
 
+/**
+ * The {@link GlobalConfig} object for RxJS. It is used to configure things
+ * like how to react on unhandled errors.
+ */
+export const config: GlobalConfig = {
+  onUnhandledError: null as ((err: any) => void) | null,
+
+  onStoppedNotification: null as ((notification: ObservableNotification<any>, subscriber: Subscriber<any>) => void) | null,
+
+  Promise: undefined as PromiseConstructorLike | undefined,
```

Comment:
nitpick: now that we're using a type, we can remove these type assertions:
```suggestion
  onUnhandledError: null,
  onStoppedNotification: null,
  Promise: undefined,
```

Reply:
Yeah, noticed myself. Removed.

Reply:
Spaces between properties also?

Reply:
The empty lines? Yeah, they were really only there because of the TSDoc. They serve no real purpose, but no big deal.

Reply:
I'll fix it anyway 🙂 Thanks!